rest/offer_types.php

```diff
@@ -17,13 +17,14 @@
 
     $offer_type->name = $data['name'];
     $offer_type->category_id = $data['category_id'];
+    // $offer_type->category_name = $data['category_name']; // This puts it in but isn't the same format as other records
     $offer_type->default_text = $data['default_text'];
     $offer_type->active = $data['active'];
 
     if(isset($data['id'])){
 		$offer_type->id=$data['id'];
 		$offer_type->update();
-		$offer_type->read();
+		//$offer_type->read();
```

Comment:
Is there a reason for no longer reading it? Often we read after an update to get the values in the last_update_date and last_updated_by columns

Reply:
The read was throwing errors along the lines of `Trying to access array offset on value of type bool in ...`

Reply:
sounds like we should be looking at why the read is a problem instead of commenting it out

Reply:
I un-commented this and it didn't throw any errors so I don't think it is throwing an error anymore.